NodeBase/Dockerfile

```diff
@@ -112,16 +112,31 @@ RUN  wget -nv -O noVNC.zip \
 #=========================================================================================================================================
 RUN chmod +x /dev/shm
 
+# Creating base directory for Xvfb
+RUN mkdir -p /tmp/.X11-unix && \
+    fix-permissions /tmp/.X11-unix
+
+#==============================
+# Generating the VNC password as seluser
+# So the service can be started with seluser
```

Comment:
But `seluser` is not being used here. I see that there is a `chown` now, perhaps this comment needs an update.

Reply:
Thank you for your review. I have updated a proper comment